src/main/kotlin/io/github/mojira/arisa/modules/privacy/PrivacyModule.kt

```diff
@@ -99,37 +99,37 @@ class PrivacyModule(
                 val redacted = if (it.uploader?.isBotUser?.invoke() == true) null else attachmentRedactor.redact(it)
                 if (redacted == null) {
                     // No redaction necessary / possible; check if attachment contains sensitive data
-                    if (!attachmentContainsSensitiveData) {
+                    if (!foundNonRedactableSensitiveData) {
                         containsSensitiveData(it.getTextContent())?.let { matchResult ->
-                            attachmentContainsSensitiveData = true
+                            foundNonRedactableSensitiveData = true
                             logFoundSensitiveData("in attachment with ID ${it.id}", matchResult)
                         }
                     }
                     return@mapNotNull null
-                } else {
```

Comment:
Changed this from:
```
else {
  if (...) {
    return ...
  }
  return
}
```
to
```
else if (...) {
  return ...
} else {
  return
}
```

These are (except for the variable name change) only indentation changes.

Reply:
As this is Kotlin and if statements are expressions, in theory you should be able to put the `return` in front of the `if` too, I believe.

Reply:
In theory the `return@mapNotNull` could be removed as well, but at least at the moment I am not convinced by that Kotlin feature yet, at least for larger code blocks as it is the case here. In my opinion that looks too similar to an unused expression (which Kotlin allows, but at least IntelliJ warns about it).